_layouts/default.html

```diff
@@ -20,5 +21,16 @@
       For suggestions, requests, and the like,
       see the <a href="{{ site.github.repository_url }}">GitHub repository</a>.
     </footer>
+
+    {% if page.has_clips %}
+    <div id="player" class="player-hidden">
+      <div id="player-controls">
+        <button id="player-closebtn">✕</button>
+      </div>
+      <div id="player-embed-container"> </div>
+    </div>
+    <script src="{{ "/assets/js/player.js" | relative_url }}"></script>
+    {% endif %}
+    <script src="{{ "/assets/js/app.js" | relative_url }}"></script>
```

Comment:
The only thing `app.js` is doing at the moment is controlling details unfolding, which is only relevant to the Home page.